ui/app/services/capabilities.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Service, { service } from '@ember/service';
```

Comment:
This is a new pattern, I liked the idea of consolidating the capabilities logic. I can also pull this into a separate PR to main if folks agree with this.